src/model/job-advert.v1.yaml

```diff
@@ -0,0 +1,45 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Job advert
+type: object
+allOf:
+  - $ref: ../snippets/job-advert.v1.yaml
+  - properties:
+        description:
+            description: Description
+            type: array
+            items:
+                $ref: ../misc/blocks-all.v1.yaml
+            minItems: 1
+        responsibilities:
```

Comment:
There is a trade-off between a fixed schema and consistency. It seems most of the job adverts in the past cloned an existing one, tweaking the sections, but they were very similar for being generic blog posts. Still it's important to be able to customize in the future so some of these can be cut away in favor of the generic `content`, which we need anyway.